prime-router/src/main/resources/db/migration/V6__Create_setting_tables.sql

```diff
@@ -0,0 +1,43 @@
+/*
+This SQL creates the tables of the DB. The Flyway tool applies this migration to create the database
+
+Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+use VARCHAR(63) for names in organization and schema
+
+Copy a version of this comment into the next migration
+*/
+
+/*
+Settings are kept a table where the actual settings values are in a JSON field. Receivers and Senders
+have a parent organizationId. Organizations do not have a setting.
+The setting is versioned to keep around old values for lineage auditing. Deleted settings
+are tomb stoned, a row is never deleted.
+*/
+CREATE TYPE SETTING_TYPE AS ENUM ('ORGANIZATION', 'RECEIVER', 'SENDER');
+CREATE TABLE setting (
+    -- Key
+    setting_id SERIAL PRIMARY KEY,
+    type SETTING_TYPE NOT NULL,
+    name VARCHAR(63) NOT NULL,
+    organization_id INT NULL REFERENCES setting(setting_id),
+
```

Comment:
@jimduff-usds A couple of your suggestions are incorporated here, including this FK constraint.